installer/rpm/algorand.spec

```diff
@@ -131,10 +102,12 @@ getent group nogroup >/dev/null || \
 
 %post
 chown -R algorand /var/lib/algorand
-%systemd_post algorand.service
+%systemd_post algorand
```

Comment:
should we do `systemd_post algorand@*` too?

Reply:
I don't think that will have any effect. I think plain 'post' only runs after the initial install but the preun-postun run on both uninstall and upgrade's stop-start. I wouldn't expect anyone to have created any systemd algorand@... instances before the initial install